sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -143,17 +143,54 @@ public EditEventSteps(
           webDriverHelpers.clickOnWebElementBySelector(POPUP_SAVE);
           if (webDriverHelpers.isElementVisibleWithTimeout(PICK_OR_CREATE_PERSON_POPUP, 15)) {
             webDriverHelpers.clickOnWebElementBySelector(CREATE_NEW_PERSON_RADIO_BUTTON);
+            webDriverHelpers.scrollToElement(PICK_OR_CREATE_POPUP_SAVE_BUTTON);
             webDriverHelpers.clickOnWebElementBySelector(PICK_OR_CREATE_POPUP_SAVE_BUTTON);
           }
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(
               PERSON_DATA_ADDED_AS_A_PARTICIPANT_MESSAGE);
           person = collectPersonUuid();
           selectResponsibleRegion("Region1");
           selectResponsibleDistrict("District11");
-          webDriverHelpers.clickOnWebElementBySelector(POPUP_SAVE);
+          //     webDriverHelpers.clickOnWebElementBySelector(POPUP_SAVE);
+          webDriverHelpers.scrollToElement(PICK_OR_CREATE_POPUP_SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(PICK_OR_CREATE_POPUP_SAVE_BUTTON);
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(PERSON_DATA_SAVED);
         });
 
+    When(
```

Comment:
for all these 3 methods below, let's do the following because we have a task created to split saving logic from steps in order to make them reusable.
1. Save action we want to keep it separated, because in some situations we don't want to save, and can be reused in feature file instead of adding it in step logic. So create a method only for the save action.
2.  Let's create also an UI POJO for event participant with all fields (checked and looks like we don't have one, and well need it in future for other tests)
3. For last 2 methods, since we'll move out the save logic outside, and we'll keep only the fill a specific field with specific data, so we can move out also the error message validation (and close action)
- this would help to determine failures in allure, because keeping the full login in a single method, will be confusing to determine exactly which step has failed, and will make the code reusable for multiple fields within view/popup

Reply:
1. OK
2. There is a method with all fields - "I add a participant to the event"
3. I can split  every check error messages but i can not parameterize it in that test scenario, it will be extra methods in the test

Reply:
2. I know, but we needed to refactor that approach to use it also in further tests. If you think is out of scope for your PR, let's discuss and create a task to refactor this after
3. Ok, no worries, I'm happy with this approach